server/services/BpmnToSsotParsing/BpmnToSsotParsing.jsx

```diff
@@ -0,0 +1,144 @@
+/* eslint-disable no-underscore-dangle */
+
+/**
+ * @category Server
+ * @module
+ */
+
+/**
+ * @description Creates a base element of the single source of truth
+ * @returns {object}  Base element of the single source of truth
+ */
+const createBaseElement = (id) => {
+    const baseElement = {
+        "type": "",
+        "name": "",
+        "id": id,
+        "predecessorIds": [],
+        "successorIds": []
+    }
+    return baseElement
+};
+
+/**
+ * @description Creates the metadata object of the single source of truth
+ * @returns {object}  Metadata object of the single source of truth
+ */
+const createMetadataObj = () => {
+    const metadataObj = {
+        "robotId": "exampleRobot",
+        "starterId": "exampleID"
+    }
+    return metadataObj
+};
+
+/**
+ * @description Checks if the given id can be found in the given Array of element objects
+ * @returns {boolean}  Boolean if element is tracked in Array
+ */
+const isElementTracked = (elementsArray, id) => {
```

Comment:
How about `return elementsArray.find(element => (element.id === id))` ?

Reply:
I tried this, but this does not work. If the element was found, it would return it instead of true. See [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find):

> The value of the first element in the array that satisfies the provided testing function. Otherwise, undefined is returned.